package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "bignumber.js": "^9.1.1",
+    "chalk": "^4.1.2",
```

Comment:
We try to minimize numer of dependencies, we are adding deps only if there is no easy way to do it without dep

Reply:
We would have to build this ourselves or inline ansi-escapes. Chalk is very lightweight and industry standard for this task.

Thanks for the heads up about dependencies ❤️